03_letters/CameronSchafer/letters_test.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "a:2\nb:1\n"
+	actual := buf.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+// sortLetters()
+func TestSortLetters(t *testing.T) {
+	// test cases with descriptions.
+	testCases := []struct {
```

Comment:
You can make testCases a global var and do all the tests, (table tests)
Also might be good to have multiple global vars to test a set of edge cases, ex 
`testWhitespaces` 
`testSpecials`

Reply:
This file was removed because wrong lab